src/app/hooks/useOptimizelyVariation/useServerSide/index.test.tsx

```diff
@@ -59,7 +78,10 @@ describe('useOptimizelyMvtVariation custom hook', () => {
         enabled: true,
       },
     ];
-    const result = renderUseOptimizelyMvtVariation(mockMvtExperiments, 'bar');
+    const result = renderUseServerSide({
+      serverSideExperiments: mockMvtExperiments as ServerSideExperiment[],
```

Comment:
Could we also rename this variable? 
```suggestion
      serverSideExperiments: mockServerSideExperiments as ServerSideExperiment[],
```